corehq/apps/reports/tasks.py

```diff
@@ -370,9 +370,9 @@ def _expose_download(fpath, use_transfer, zip_name, download_id, num_forms):
 
 
 def _format_filename(form_info, question_id, extension, case_id_to_name):
-    filename = u"{}-user_{}-form_{}{}".format(
+    filename = u"{}-{}-form_{}{}".format(
         unidecode(question_id),
-        form_info['form'].user_id or 'unknown',
+        form_info['form'].get_data('form/meta/username') or form_info['form'].user_id or 'user_unknown',
```

Comment:
ah yes we do fetch the form elsewhere. i had thought we did aggressive caching of the form.xml since it's pretty much a read only thing, but just looked and it doesn't seem like we don't cache beyond `memoized`. `memoized` in this case should be enough since we don't re-fetch form, but i'll still move it closer to the original call to get_xml